Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -214,13 +214,13 @@ private static bool IsFileMask(string input)
             var b = false;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L168
This method can be `static`.

https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L194
This method can be `static`.